ui/src/app/components/help-sidebar.tsx

```diff
@@ -334,7 +340,7 @@ export const HelpSidebar = fp.flow(
           faIcon: faDna,
           label: 'Genomic Extraction',
           showIcon: () => true,
-          style: {height: '22px', width: '22px', marginTop: '0.25rem'},
+          style: {height: '22px', width: '22px', marginTop: '0.25rem', position: 'absolute'},
```

Comment:
This is the same thing we do at L550 to make sure the status icon doesn't push the runtime icon to the left. Will comment

Reply:
the difference here is that we're using FontAwesome to display this one instead of an SVG asset, and the function we use to display FA icons takes the entire icon config with label, style, tooltip etc.